experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/unboxed.spec.ts

```diff
@@ -211,6 +279,254 @@ describe("unboxed unit tests", () => {
 				assert.equal(unboxed.value, true);
 			});
 		});
+
+		describe("Value field", () => {
+			it("Boolean", () => {
+				const { fieldSchema, context, cursor } = createValueLeafTree(
+					ValueSchema.Boolean,
+					true,
+				);
+				assert.equal(unboxedField(context, fieldSchema, cursor), true);
+			});
+
+			it("Number", () => {
+				const { fieldSchema, context, cursor } = createValueLeafTree(
+					ValueSchema.Number,
+					42,
+				);
+				assert.equal(unboxedField(context, fieldSchema, cursor), 42);
+			});
+
+			it("String", () => {
+				const { fieldSchema, context, cursor } = createValueLeafTree(
+					ValueSchema.String,
+					"Hello world",
+				);
+				assert.equal(unboxedField(context, fieldSchema, cursor), "Hello world");
+			});
+
+			// TODO: Fluid Handle
+
+			it("Struct", () => {
```

Comment:
From the name of this test, I can't tell if this is testing that value fields containing a union of a single struct type unwrap properly, of if this is testing a struct's value field unwrap properly. I don't think either of those belong in field fieldUnboxing tests either.

Reply:
More specifically: which part of the code in unboxedField is this testing? The field kind check for value? This is a lot of code for that, and thats covered by other tests.

Reply:
I've reduced the cases based on your feedback. This case has been removed.